pkg/util/tests/tests.go

```diff
@@ -49,11 +50,21 @@ func GenerateRuntime(cnf *config.Control) error {
 	os.MkdirAll(filepath.Join(cnf.DataDir, "tls"), 0700)
 	os.MkdirAll(filepath.Join(cnf.DataDir, "cred"), 0700)
 
-	deps.FillRuntimeCerts(cnf, runtime)
+	deps.CreateRuntimeCertFiles(cnf, runtime)
 
 	if err := deps.GenServerDeps(cnf, runtime); err != nil {
 		return err
 	}
 	cnf.Runtime = runtime
 	return nil
 }
+
+func ClusterIPNet() *net.IPNet {
+	_, clusterIPNet, _ := net.ParseCIDR("10.42.0.0/16")
+	return clusterIPNet
+}
+
+func ServiceIPNet() *net.IPNet {
+	_, serviceIPNet, _ := net.ParseCIDR("10.43.0.0/16")
+	return serviceIPNet
+}
```

Comment:
what are these used for?

Reply:
Used in another test that I found I didn't need but useful nonetheless. Code updated to use them where appropriate. I will use them again in a future PR.